src/systemctl/systemctl.c

```diff
@@ -3180,12 +3184,18 @@ static int start_unit(int argc, char *argv[], void *userdata) {
                         return log_error_errno(r, "Failed to attach bus to event loop: %m");
         }
 
+        if (method_is_stop)
+                names_stopped = newa0(char*, strv_length(names) + 1);
```

Comment:
i am pretty sure this will trip up static checkers sooner or laters, because they are unlikely to understand that names_stopped not being initialized doesn't matter if "method_is_stop" is not true